data/styles/default/include/Icons.mapcss

```diff
@@ -2125,8 +2125,8 @@ area|z16[shop=tattoo]
 node|z16[shop=erotic],
 area|z16[shop=erotic]
 {icon-image: erotic-m.svg;}
-node|z16[shop=funeral_directors],
-area|z16[shop=funeral_directors]
+node|z18[shop=funeral_directors],
+area|z18[shop=funeral_directors]
```

Comment:
you can delete the lines here completely,
because further there is another section
```
node|z17-[shop=funeral_directors],
area|z17-[shop=funeral_directors]
{icon-image: funeral_directors-m.svg;icon-min-distance: 24;}
```
you have to change it to z18- and remove the `icon-min-distance: 24;`

And if you want, you can remove redundant `area|` lines from the POIs you've touched too